spec/controllers/videos_controller_spec.rb

```diff
@@ -0,0 +1,28 @@
+RSpec.describe VideosController, type: :controller do
+  describe 'GET show' do
+    context 'when a video does exist' do
+      let(:categories) { [] }
+      let(:video) { Core::Video.new('test', title: 'test', body: 'body', categories: categories) }
+
+      before do
+        expect(Core::VideoReader).to receive(:new) do
```

Comment:
This should be `allow`, the difference is subtle but meaningful in terms of what we are trying to express in the spec. The `before` being for setup, rather than specifying expectations on behaviour.